python/lsst/ts/xml/data/sal_interfaces/Electrometer/Electrometer_Events.xml

```diff
@@ -307,4 +307,65 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>Electrometer</Subsystem>
+    <EFDB_Topic>Electrometer_logevent_fileSummary</EFDB_Topic>
```

Comment:
This should be scanSummary since this is a summary of the scan and should not imply a relationship with the fits file uploaded to the LFA as that may confuse people.